internal/common/sharedfontdb/fontconfig.rs

```diff
@@ -3,51 +3,105 @@
 
 #![allow(unsafe_code)]
 
-use fontconfig_sys as ffi;
+use core::ffi::{c_char, c_int, c_uchar, c_void};
 
-use ffi::statics::LIB;
+const FC_MATCH_PATTERN: c_int = 0;
+const FC_RESULT_MATCH: c_int = 0;
+
+#[repr(C)]
+struct FcFontSet {
+    nfont: c_int,
+    sfont: c_int,
+    fonts: *mut *mut c_void,
+}
 
 // This is duplicated in the slint-compiler's glyph embedding code
 pub fn find_families(requested_family: &str) -> Vec<String> {
     unsafe {
-        let config = (LIB.FcInitLoadConfigAndFonts)();
+        let fontconfig = libloading::Library::new("libfontconfig.so.1")
+            .expect("Unable to dlopen libfontconfig.so.1");
```

Comment:
Can we maybe not panic here in this case?
Returning an error instead?

Reply:
I can try to do that, yes. The panic will then be elsewhere :-). I'll post a follow-up - then you can decide if you prefer this or the other option.